payments-core/api/payments-core.api

```diff
@@ -6490,6 +6490,14 @@ public final class com/stripe/android/networking/FraudDetectionData$Creator : an
 	public synthetic fun newArray (I)[Ljava/lang/Object;
 }
 
+public final class com/stripe/android/networking/StripeRepository$DefaultImpls {
```

Comment:
Why this change?

Reply:
Because `StripeRepository` is no longer an abstract class, but an interface.